packages/babel-parser/src/plugins/flow/index.js

```diff
@@ -1059,13 +1060,9 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         } else {
           let kind = "init";
 
-          if (this.isContextual("get") || this.isContextual("set")) {
+          if (this.isContextual(tt._get) || this.isContextual(tt._set)) {
             const lookahead = this.lookahead();
-            if (
-              lookahead.type === tt.name ||
-              lookahead.type === tt.string ||
-              lookahead.type === tt.num
-            ) {
+            if (tokenIsLiteralPropertyName(lookahead.type)) {
```

Comment:
This is a bug fix: Flow allows any identifiers to be the accessor type property. e.g.
```js
type A = { get if(): number }
```
is a [valid Flow code](https://flow.org/try/#0C4TwDgpgBAglC8UDeUDmFhQJYDMAUAlAFxQB2ArgLYBGEATlAL5A).